### PR TITLE
feat: Point cloud tool view optimizations

### PR DESCRIPTION
**Pull request by lixinghua123**

### CODE REVIEW

File: packages/lb-components/src/components/pointCloudView/components/sideAndBackOverView/index.tsx [hunk 1]
```diff
@@ -50,40 +50,41 @@ const SideAndBackOverView = (props: IProps) => {
...
         [getClassName('point-cloud-container', 'left-bottom')]: true,
-        [getClassName('point-cloud-container', 'left-bottom-float')]: selectAndEnlarge,
+        [getClassName('point-cloud-container', 'left-bottom-float')]: true,
       })}
       style={{
         top: position.y,
         left: position.x,
         width: 360,
+        height: 400,
+        opacity: selectAndEnlarge ? 1 : 0,
```
Comment (Kerwin-L) [inline packages/lb-components/src/components/pointCloudView/components/sideAndBackOverView/index.tsx:60]: wrong way.

**Review by lihqi — APPROVED** [review]

**Pull request merged**